lib/entities/request/DutchRequest.ts

```diff
@@ -43,38 +45,16 @@ export class DutchRequest implements QuoteRequest {
       {
         swapper: body.swapper ?? NATIVE_ADDRESS,
         exclusivityOverrideBps: body.exclusivityOverrideBps ?? DEFAULT_EXCLUSIVITY_OVERRIDE_BPS.toNumber(),
-        auctionPeriodSecs: body.auctionPeriodSecs ?? DutchRequest.defaultAuctionPeriodSecs(info.tokenInChainId),
-        deadlineBufferSecs: body.deadlineBufferSecs ?? DutchRequest.defaultDeadlineBufferSecs(info.tokenInChainId),
+        startTimeBufferSecs: body.startTimeBufferSecs,
+        auctionPeriodSecs: body.auctionPeriodSecs,
+        deadlineBufferSecs: body.deadlineBufferSecs,
         useSyntheticQuotes: body.useSyntheticQuotes ?? false,
       }
     );
   }
 
   constructor(public readonly info: DutchQuoteRequestInfo, public readonly config: DutchConfig) {}
 
-  // TODO: parameterize this based on other factors
-  public static defaultAuctionPeriodSecs(chainId: number): number {
```

Comment:
not not doing defaults at the request level anymore because sometimes info from the _quote_ might impact it (i.e. whether its open quote or exclusive quote)